src/Http/routes.php

```diff
@@ -17,6 +17,9 @@ function () {
         Route::get('login', 'SecurityController@showLogin')->name($p.'login.get');
         Route::post('login', 'SecurityController@login')->name($p.'login.post');
         Route::get('logout', 'SecurityController@logout')->name($p.'logout')->middleware($auth);
+        Route::get('p3/login', 'SecurityController@showLogin')->name($p.'login.get');
+        Route::post('p3/login', 'SecurityController@login')->name($p.'login.post');
+        Route::get('p3/logout', 'SecurityController@logout')->name($p.'logout')->middleware($auth);
```

Comment:
`name($p.'login.get')` should be renamed as `name($p.'v3.login.get')`, respectively.